tools/opt/LLVMBuild.txt

```diff
@@ -23,12 +23,18 @@ required_libraries =
  AsmParser
  BitReader
  BitWriter
+ DXIL
+ HLSL
```

Comment:
This will work I'm sure, but the fact is that LLVMHLSL depends on LLVMDXIL. This solves the current problem, but leaves the pitfall around for the next thing that tries to depend on HLSL and not DXIL.

To solve the bigger problem, we might add DXIL as a dependency for HLSL.